en/docs/concepts/user-management/realm.md

```diff
@@ -0,0 +1,51 @@
+# Introduction to Realms
+
+The complete functionality and contents of the User Management module is called a **user realm**. The realm includes the user management
+classes, configurations and repositories that store information. Configuring the User Management functionality in WSO2 Identity Server involves setting up the relevant repositories and updating the relevant configuration files.
+
+---
+
+## Configuring the authorization manager
+
```

Comment:
the subheading talks about an authorization manager, but the content doesn't make it clear what an authorization manager is or what it does